crypto/fipsmodule/sha/sha3.c

```diff
@@ -377,3 +377,48 @@ int SHAKE_Squeeze(uint8_t *md, KECCAK1600_CTX *ctx, size_t len) {
   //FIPS_service_indicator_update_state();
   return 1;
 }
+
+int SHAKE128_Init_x4(KECCAK1600_CTX_x4 *ctx) {
+
+  int ok = (SHAKE_Init(&(*ctx)[0], SHAKE128_BLOCKSIZE) &&
```

Comment:
If it is intentional that later calls are skipped if earlier calls fail, this may be worth documenting. Same for the othe functions.